apps/src/block_utils.js

```diff
@@ -1102,34 +1102,43 @@ exports.createJsWrapperBlockCreator = function(
           this.initMiniFlyout(miniToolboxXml);
         }
 
-        // For mini-toolbox, indicate which blocks should receive the duplicate on drag
-        // behavior and indicates the sibling block to shadow the value from
-        if (this.type === 'gamelab_clickedSpritePointer') {
-          this.setParentForCopyOnDrag('gamelab_spriteClickedSet');
```

Comment:
note: I don't think this function does anything anymore. Going to open a blockly PR to delete it.